explorer/app/theme/theme.ts

```diff
@@ -0,0 +1,98 @@
+import { createTheme } from "@mui/material/styles";
+
+export const theme = createTheme({
+  spacing: 4,
+  typography: {
```

Comment:
I think the latest style guide has been updated - let's update the typography theme to match the typography styles [here](https://www.figma.com/file/i4uZsYkKAML9ivi2TIo6ug/explore-dashboard?node-id=470%3A4666).

I recommend we completely remove from the theme any typography variants we will not be using or have not yet been defined e.g. `caption` or `overline` or even `body1` so that there is no confusion later on - if and when we do want those styles to be used we know that we are creating them from scratch and that we aren't "modifying" a style.

@MillenniumFalconMechanic can you please confirm whether or not we should match the style guide naming convention e.g. `text-heading-xlarge` and so on? And @BruceRodrigues would it possible to match the style guide naming convention e.g. `my-typography-variant` as apposed to `myTypographyVariant`? <-- wondering if this is technically achievable?

Reply:
Sure, that's possible!